lib/Runtime/Types/PathTypeHandler.cpp

```diff
@@ -299,11 +299,90 @@ namespace Js
         return PathTypeHandlerBase::AddPropertyInternal(instance, propertyId, value, info, flags, possibleSideEffects);
     }
 
-    BOOL PathTypeHandlerBase::DeleteProperty(DynamicObject* instance, PropertyId propertyId, PropertyOperationFlags flags)
+    void PathTypeHandlerBase::MoveAuxSlotsToObjectHeader(DynamicObject *const object)
```

Comment:
>MoveAuxSlotsToObjectHeader [](start = 30, length = 26)

pls consider adding traces for this scenario. one of them could be ObjectHeaderInlining phase.